From 040c65325631d3e5faac16919b60ffd8bc8e0c3e Mon Sep 17 00:00:00 2001
From: effectsbot <eff3ctsb0t@gmail.com>
Date: Sun, 15 Feb 2026 12:22:10 +0100
Subject: [PATCH 10/11] wine.inf: Disable unused services

Signed-off-by: effectsbot <eff3ctsb0t@gmail.com>
---
 loader/wine.inf.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 24a7cc5..23bb51b 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -2653,7 +2653,6 @@ StartType=3
 ErrorControl=1
 
 [Services]
-HKLM,%CurrentVersion%\RunServices,"winemenubuilder",2,"%11%\winemenubuilder.exe -a -r"
 HKLM,"System\CurrentControlSet\Services\Eventlog\Application",,16
 HKLM,"System\CurrentControlSet\Services\Eventlog\System","Sources",0x10000,""
 HKLM,"System\CurrentControlSet\Services\Tcpip\Parameters","DataBasePath",,"%12%\etc"
@@ -2773,10 +2772,8 @@ services,"@%11%\ws2_32.dll,-4"
 [InfFiles]
 input.inf,"@%12%\hidclass.sys,-1"
 mouhid.inf,"@%12%\mouhid.sys,-1"
-winebth.inf,"@%12%\winebth.sys,-1"
 winebus.inf,"@%12%\winebus.sys,-1"
 winehid.inf,"@%12%\winehid.sys,-1"
-wineusb.inf,"@%12%\wineusb.sys,-1"
 winexinput.inf,"@%12%\winexinput.sys,-1"
 
 [NlsFiles]
-- 
2.53.0

