From 61cd85a7e3928d5b9ad2171b32223f2a416f8307 Mon Sep 17 00:00:00 2001
From: effectsbot <eff3ctsb0t@gmail.com>
Date: Sun, 15 Feb 2026 12:18:00 +0100
Subject: [PATCH 12/12] wine.inf: Add Kombucha-specific registry overrides

Signed-off-by: effectsbot <eff3ctsb0t@gmail.com>
---
 loader/wine.inf.in | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 23bb51b2053..f7f689863ed 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -63,6 +63,7 @@ AddReg=\
     Debugger,\
     DirectX,\
     Fonts,\
+    KombuchaOverrides,\
     MCI,\
     Misc,\
     OLE,\
@@ -87,6 +88,7 @@ AddReg=\
     Debugger,\
     DirectX,\
     Fonts,\
+    KombuchaOverrides,\
     MCI,\
     Misc,\
     OLE,\
@@ -114,6 +116,7 @@ AddReg=\
     Debugger,\
     DirectX,\
     Fonts,\
+    KombuchaOverrides,\
     MCI,\
     Misc,\
     OLE,\
@@ -141,6 +144,7 @@ AddReg=\
     Debugger,\
     DirectX,\
     Fonts,\
+    KombuchaOverrides,\
     MCI,\
     Misc,\
     OLE,\
@@ -164,6 +168,7 @@ AddReg=\
     CurrentVersionWow64.ntx86,\
     Debugger,\
     DirectX,\
+    KombuchaOverrides,\
     MCI,\
     Misc,\
     Packages.wow64,\
@@ -182,6 +187,7 @@ AddReg=\
     CurrentVersionWow64.ntarm,\
     Debugger,\
     DirectX,\
+    KombuchaOverrides,\
     MCI,\
     Misc,\
     Tapi,\
@@ -2866,3 +2872,17 @@ EtcFiles  = 12,etc
 InfFiles  = 17
 NlsFiles  = 11
 SortFiles = 10,globalization\sorting
+
+[KombuchaOverrides]
+;; Global overrides
+HKCU,Software\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Layers,"",,"~ HIGHDPIAWARE"
+;; Studio-specific overrides
+HKCU,Software\Wine\AppDefaults\RobloxStudioBeta.exe\DllOverrides,"msvcp140",,"native"
+;; WebView2-specific overrides
+HKCU,Software\Wine\AppDefaults\msedgewebview2.exe,"Version",,"win8"
+HKCU,Software\Wine\AppDefaults\msedgewebview2.exe\DllOverrides,"d3d8",,"builtin"
+HKCU,Software\Wine\AppDefaults\msedgewebview2.exe\DllOverrides,"d3d9",,"builtin"
+HKCU,Software\Wine\AppDefaults\msedgewebview2.exe\DllOverrides,"d3d10core",,"builtin"
+HKCU,Software\Wine\AppDefaults\msedgewebview2.exe\DllOverrides,"d3d11",,"builtin"
+HKCU,Software\Wine\AppDefaults\msedgewebview2.exe\DllOverrides,"dxgi",,"builtin"
+HKCU,Software\Wine\Fonts\Replacements,"Arial",,"Liberation Sans"
-- 
2.53.0

