diff --git a/programs/wineboot/wineboot.c b/programs/wineboot/wineboot.c
index e8bbdcf..025bb31 100644
--- a/programs/wineboot/wineboot.c
+++ b/programs/wineboot/wineboot.c
@@ -1512,7 +1512,10 @@ static HWND show_wait_window(void)
 {
     HWND hwnd = CreateDialogParamW( GetModuleHandleW(0), MAKEINTRESOURCEW(IDD_WAITDLG), 0,
                                     wait_dlgproc, (LPARAM)prettyprint_configdir() );
-    ShowWindow( hwnd, SW_SHOWNORMAL );
+    WCHAR *hide = _wgetenv( L"WINEBOOT_HIDE_DIALOG" );
+    if (hide == NULL)
+    	ShowWindow( hwnd, SW_SHOWNORMAL );
+
     return hwnd;
 }
 
